galasa-extensions-parent/dev.galasa.ras.couchdb/src/main/java/dev/galasa/ras/couchdb/internal/CouchdbRasStore.java

```diff
@@ -39,14 +39,20 @@
 import dev.galasa.framework.spi.teststructure.TestStructure;
 import dev.galasa.framework.spi.utils.GalasaGson;
 import dev.galasa.extensions.common.api.HttpClientFactory;
+import dev.galasa.extensions.common.api.LogFactory;
+import dev.galasa.extensions.common.couchdb.pojos.PutPostResponse;
 import dev.galasa.extensions.common.impl.HttpClientFactoryImpl;
+import dev.galasa.extensions.common.impl.HttpRequestFactoryImpl;
+import dev.galasa.extensions.common.api.HttpRequestFactory;
 import dev.galasa.extensions.common.impl.LogFactoryImpl;
 import dev.galasa.ras.couchdb.internal.pojos.Artifacts;
 import dev.galasa.ras.couchdb.internal.pojos.LogLines;
-import dev.galasa.ras.couchdb.internal.pojos.PutPostResponse;
 
 public class CouchdbRasStore implements IResultArchiveStoreService {
 
+    private static final String COUCHDB_AUTH_ENV_VAR = "GALASA_RAS_TOKEN";
```

Comment:
GALASA_COUCHDB_TOKEN instead of GALASA_RAS_TOKEN ? and call the constant the same thing ? Confusion between couchdb's authentication mechanism and our auth store ?